MANIFEST.IN

```diff
@@ -0,0 +1 @@
+include galaxy_ng/app/webserver_snippets/*
```

Comment:
@cutwater got rid of our manifest. If you're re adding it, please correct the name from `MANIFEST.IN` to `MANIFEST.in` we had a typo in our original filename.